.github/workflows/haskell.yml

```diff
@@ -95,12 +95,10 @@ jobs:
     - name: Install secp256k1 (Windows)
       if: matrix.os == 'windows-latest'
       run: |
-        mkdir secp256k1-sources
-        cd secp256k1-sources
-        git clone https://github.com/bitcoin-core/secp256k1.git
-        cd secp256k1
-        git reset --hard $SECP256K1_REF
-        echo "FIXME: install from sources"
+        echo "FIXME: can't install from sources, so instead libsecp256k1 support is disabled"
+        cat >> cabal.project <<EOF
+        package cardano-crypto-class
```

Comment:
I think it is better to do `cabal configure --flags="-secp256k1-support"'` instead of editing cabal.project

Reply:
This is what is done further down for configuring libsodium, so I just copied that :shrug:

Reply:
I also don't think that multiple calls to `cabal configure` will compose in the way we want.